include/deal.II/lac/scalapack.h

```diff
@@ -189,6 +189,153 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
                const std::pair<unsigned int,unsigned int> &offset_B,
                const std::pair<unsigned int,unsigned int> &submatrix_size) const;
 
+  /**
+   * Transposing assignment: $\mathbf{A} = \mathbf{B}^T$
+   *
+   * The matrices $\mathbf{A}$ and $\mathbf{B}$ must have the same process grid.
+   *
+   * The following alignment conditions have to be fulfilled: $MB_A=NB_B$ and $NB_A=MB_B$.
+   */
+  void copy_transposed(const ScaLAPACKMatrix<NumberType> &B);
+
+  /**
+   * The operations based on the input parameter @p transpose_B and the alignment conditions are summarized in the following table:
+   *
+   * | transpose_B |          Block Sizes         |                    Operation                  |
+   * | :---------: | :--------------------------: | :-------------------------------------------: |
+   * |   false     | $MB_A=MB_B$ <br> $NB_A=NB_B$ |  $\mathbf{A} = a \mathbf{A} + b \mathbf{B}$   |
```

Comment:
in another PR https://github.com/dealii/dealii/pull/5899#discussion_r168256838 @kronbichler raised the point whether we shall use bold symbols (`\mathbf`) for matrices and vectors. 

Personally, I don't mind sticking with plain LaTeX `A`, `B`, `c` for linear algebra, probably with `\cdots` to denote multiplication. Downside is that we won't distinguish between scalars and vectors in this notation.

@dealii/dealii thoughts?